ref-filter.c

```diff
@@ -1637,6 +1651,12 @@ void show_ref_array_item(struct ref_array_item *info, const char *format, int qu
 	putchar('\n');
 }
 
+/* Wrapper: Show ref_array_item w/o referencing container in function name */
```

Comment:
Not as tag, rather as *"thin wrapper around ..."* or *"high-level wrapper for ..."* and the like.